Azure/sonic-buildimage
# [sflow] Update version of hsflowd

* Updating version of hsflow daemon to apply
  fix, which resolves problem of switching
  between IPv4 and IPv6, in case when the
  IPv4 has deleted for the interface.

Signed-off-by: Maksym Belei <Maksym_Belei@jabil.com>



#### Why I did it
Resolves https://github.com/Azure/sonic-buildimage/issues/6951
The new release of `hsflowd `contains the fix for the issue: https://github.com/sflow/host-sflow/commit/2703ecb173bdbc6b1bc6cdea524eafa9640f141e

#### How I did it
`HSFLOWD_VERSION `env variable has changed in the rules to be pointed to the latest release of `hsflowd`.

#### How to verify it
1. `sudo config sflow enable`
2. `sudo config loopback add Loopback1`
`sudo config int ip add Loopback1 a84f:97ff:fea7:33a5::fe80/64`
`sudo config int ip add Loopback1 192.168.101.1/24`
2. `sudo config sflow agent-id add Loopback1`
`sudo config sflow collector add Collector1 192.168.101.1`
`sudo config sflow collector add Collector2 a84f:97ff:fea7:33a5::fe80`
3. use `sudo sflowtool -p 6343 -l` for checking sflow data
4. remove and add again the ipv4 entry of Loopback1.
`hsflowd `should change agent ip from IPv4 to IPv6 and wise versa, depending on IPv4 entry present or not.
Switching between IPs is being performed by `hsflowd`, based on IP address priority ranking.


#### Which release branch to backport (provide reason below if selected)




The issue has reported on 202012, but is reproducible on master branch too.

#### Description for the changelog

Update of hsflowd to the latest release version.
